advantageswervesomeworking/src/main/java/frc/robot/PortMap.java

```diff
@@ -3,10 +3,37 @@
 public class PortMap {
 
   // TODO set correct ports
-  public static final int INTAKE_MASTER_PORT = 7;
-  public static final int INTAKE_SLAVE_PORT = 0;
-  public static final int INTAKE_SENSOR_PORT = 0; // DIO port
+  public class Intake {
 
-  public static final int SHOOTER_MASTER_PORT = 9;
-  public static final int SHOOTER_SLAVE_PORT = 8;
+    public static final int INTAKE_MASTER_PORT = 7;
+    public static final int INTAKE_SLAVE_PORT = 0;
+    public static final int INTAKE_SENSOR_PORT = 0; // DIO port    
```

Comment:
Może dopisać które to pwm, które can itp. żeby przy każdej zmiennej był co to za port dla bażantów ;P